.pre-commit-config.yaml

```
@@ -1,7 +1,7 @@
 default_install_hook_types: [pre-push]
 repos:
 -   repo: https://github.com/SonarSource/sonar-secrets-pre-commit
-    rev: v2.30.0.8284
+    rev: v2.32.0.9670
```

What about avoiding the problem altogether?
```suggestion
    rev: master
```
Or better, ask them to provide `v1` branch 
```suggestion
    rev: v1
```

---

Unfortunately [not possible](https://sonarsource.slack.com/archives/C0585H9EPCZ/p1763032116275139?thread_ts=1763022577.735619&cid=C0585H9EPCZ). Renovate is the way to go.